### DISCUSSION TITLE
Free Asset Replacement Discussion

### DISCUSSION THREAD

**Discussion by KevDoy** [SFTtech/openage#1366] [Show and tell]
I know this isn't on the top of the priority list, but I would at least get the conversation started.
<img width="560" alt="Screen Shot 2021-01-19 at 3 42 09 PM" src="https://user-images.githubusercontent.com/11251011/105133697-05984880-5aa2-11eb-9d35-68ab216c5f79.png">

<img width="343" alt="Screen Shot 2021-01-19 at 3 42 31 PM" src="https://user-images.githubusercontent.com/11251011/105133701-07faa280-5aa2-11eb-8621-15fe03375541.png">

EDIT: Note the grass texture in the photo amongst the others. This is what I came up with today. 

I am experimenting with grass textures. Would y'all recommend I submit them in pull requests as I create them or wait until I have a few together?

Cheers

**Comment by TheJJ** [comment]
Yo! Awesome!
The thing with the assets is that until now we always only had a vague idea how to structure the data packs. And what formats the graphics should be in. But we're getting a clearer picture now. If you like, you can submit the pull request against the data-repo just now, so we can merge it sooner or later, but at least it doesn't get lost. If you produce more textures, you can add them to the pull you already sent then.

**Comment by heinezen** [comment]
@KevDoy The grass already looks pretty good :)

You can make a PR right now because if there is something to discuss, we can solve it once and you don't have to make a bunch of changes to all textures.

**Comment by KevDoy** [comment]
@TheJJ oh good to know. I noticed AOE2:HD uses PNG files for these tiles which is how I easily swapped them to test.
